src/index.js

```diff
@@ -15,15 +16,54 @@ function inferOption(option, defaultValue) {
   return option ? {} : defaultValue
 }
 
+/**
+ * Recursivly get the correct import order from rollup
+ * We only process a file once
+ *
+ * @param {string} id
+ * @param {Function} getModuleInfo
+ * @param {Set<string>} seen
+ */
+function getRecursiveImportOrder(id, getModuleInfo, seen = new Set()) {
+  if (seen.has(id)) {
+    return []
+  }
+
+  seen.add(id)
+
+  const result = [id]
+  getModuleInfo(id).importedIds.forEach((importFile) => {
+    result.push(...getRecursiveImportOrder(importFile, getModuleInfo, seen))
+  })
+
+  return result
+}
+
+function flatMap(array, fn) {
+  if (Array.prototype.flatMap) {
+    return array.flatMap(fn)
+  }
+
+  let result = []
```

Comment:
When running on node 10 this causes the tests to fail for me. The following seems to work though:

```
function flatMap(array, fn) {
  if (Array.prototype.flatMap) {
    return array.flatMap(fn)
  }

  return array.reduce((memo, item) => memo.concat(fn(item)), []);
}
```

Reply:
Thanks for spotting, I fixed it. I rather not use reduce as it's harder to reason with + `fn.call` mimics the same callback as flatMap would, that's why I used it.